securedrop/models.py

```diff
@@ -291,6 +291,23 @@ class InvalidUsernameException(Exception):
     """Raised when a user logs in with an invalid username"""
 
 
+class FirstOrLastNameError(Exception):
+    """Generic error for names that are invalid."""
+
+
+class InvalidNameLength(FirstOrLastNameError):
+    """Raised when attempting to create a Journalist with an invalid name length."""
+
+    def __init__(self, name):
```

Comment:
Possibly also here with a `super` with a more clear message instead of just the `__str__` below.